speech/unit_tests/test_client.py

```diff
@@ -326,16 +326,19 @@ def speech_api(channel=None):
         self.assertEqual(
             channel_args, [(creds, _gax.DEFAULT_USER_AGENT, host)])
 
-        results = sample.sync_recognize()
+        results = [i for i in sample.sync_recognize()]
 
-        self.assertEqual(len(results), 2)
+        self.assertEqual(len(results), 1)
+        self.assertEqual(len(results[0].alternatives), 2)
         self.assertEqual(results[0].transcript,
+                         results[0].alternatives[0].transcript,
                          alternatives[0]['transcript'])
         self.assertEqual(results[0].confidence,
+                         results[0].alternatives[0].confidence,
                          alternatives[0]['confidence'])
-        self.assertEqual(results[1].transcript,
+        self.assertEqual(results[0].alternatives[1].transcript,
```

Comment:
Sorry!  I'm working on vision today and my brain didn't want to flip over to speech apparently.